test/functorch/test_eager_transforms.py

```diff
@@ -2260,6 +2260,37 @@ def f(t, int_x):
         expected = torch.autograd.functional.jacobian(partial(f, int_x=3), t)
         self.assertEqual(actual, expected)
 
+    @onlyCUDA
+    def test_lstm_cuda_jacrev(self, device):
```

Comment:
I couldn't find an opinfo for lstm so opted for a simple handwritten test (based on the issue).